src/content-helper/dashboard-page/pages/traffic-boost/preview/components/preview-header.tsx

```
@@ -145,18 +149,26 @@ export const PreviewHeader = ( {
 		<div className="traffic-boost-preview-header">
 			<div className="traffic-boost-preview-info">
 				<div className="traffic-boost-preview-info-title">
-					{ activeLink?.targetPost?.title.rendered }
+					<div dangerouslySetInnerHTML={ { __html: activeLink?.targetPost?.title.rendered } } />
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)